index-metagenomics.md

```diff
@@ -1,53 +1,128 @@
 ---
 layout: galaxy-hic
 ---
-# Galaxy Metagenomics
 
-Welcome to Galaxy Metagenomics (ASaiM) -- a webserver to process, analyse and visualize Metagenomic or in general Microbiota data. 
+<img src="{{ "/assets/media/asaim_logo.png" | absolute_url }}" height="100px" alt="ASaiM logo"/> 
+
+Welcome to **Galaxy Metagenomics** ([ASaiM](http://asaim.readthedocs.io/en/latest/)) -- a webserver to process, analyse and visualize Metagenomic and Microbiota data in general. 
+
 
-![](/assets/media/asaim_logo.png)
 
 ## Get started
 
 Are you new to Galaxy, or returning after a long time, and looking for help to get started? Take <a target="_parent" href="https://metagenomics.usegalaxy.eu/tours/core.galaxy_ui">a guided tour</a> through Galaxy's user interface. 
 
-Take <a target="_parent" href="https://metagenomics.usegalaxy.eu/tours/mothur-miseq-sop">a guided tour</a> for an introduction to Galaxy and Metagenomics data analysis. This tour guides you through the 16S Microbial Analysis with Mothur tutorial on the <a target="_parent" href="http://galaxyproject.github.io/training-material/topics/metagenomics/tutorials/mothur-miseq-sop/tutorial.html">Galaxy Training Network</a>. Follow the tutorial to understand the analysis steps better or as a help which parameters are useful.
+Want to learn about metagenomics analyses? Check our [tutorials](#tutorials) or take one of our guided tour:
+
+- <a target="_parent" href="https://metagenomics.usegalaxy.eu/tours/metagenomics-general-tutorial-amplicon">Introduction of amplicon data analyses using mothur tool suite</a>
+- <a target="_parent" href="https://metagenomics.usegalaxy.eu/tours/metagenomics-general-tutorial-shotgun">Introduction to shotgun metagenomics data analyses</a>
+- <a target="_parent" href="https://metagenomics.usegalaxy.eu/tours/mothur-miseq-sop">16S Microbial Analysis with Mothur MiSeq SOP</a>
 
-Over time we will collect more tutorials in the [Galaxy Training Network](http://galaxyproject.github.io/training-material/topics/metagenomics/).
+Check also the standard but customizable [workflows](#workflows) available there. 
```

Comment:
I don't think this link will work, or we will see many different non-metagenomic workflows, no?

Reply:
It is supposed to be an internal link to the workflow section of the page. I hope it will work

Reply:
Yes, it is, but this page will not only show your shared workflows, but a lot more, isn't it.

Reply:
I thought it will not open the workflow section of Galaxy but will send to the workflow anchor below in the same page. Maybe I misunderstood

Reply:
@bebatut you're right, but I think @bgruening  means that the link in the workflow section, https://metagenomics.usegalaxy.eu/workflows/list_published shows everything. But since there's no way to link to e.g. a search term, there's no way to fix that. I've filed a bug upstream since this has annoyed me for years. https://github.com/galaxyproject/galaxy/issues/6023